routes/curations.js

```diff
@@ -6,6 +6,12 @@ const express = require('express')
 const router = express.Router()
 const utils = require('../lib/utils')
 
+router.get('/base_url', (request, response) => {
+  const repo = request.app.locals.config.curation.store.github.repo
```

Comment:
suggest this be removed in favor of returning this value in the `/pr` route response. See other comments

Reply:
@daniellandau thoughts on this? I commented similarly in the corresponding website PR.